machine/v1alpha1/types_openstack.go

```diff
@@ -130,7 +132,7 @@ type SecurityGroupFilter struct {
 	// description filters security groups by description.
 	Description string `json:"description,omitempty"`
 	// tenantId filters security groups by tenant ID.
-	// Deprecated: use projectId instead. tenantId will be ignored if projectId is set.
+	// Deprecated: tenantId is silently ignored. Use projectId instead.
```

Comment:
IMO this is a bug - we ignore it here but not for networks or subnets - but I don't think we should fix it.

Reply:
It is deprecated everywhere. I'm just updating this comment here to note that it's silently (and always) ignored rather than only ignored if projectId is set.